features/step_definitions/landing_page_steps.rb

```diff
@@ -0,0 +1,20 @@
+When(/^I visit the LandingPage$/) do
+  landing_page.load
+end
+
+Then(/^I want to be able to input my age$/) do
+  expect(page).to have_css('#your_details_form_age')
+end
+
+And(/^I want to be able to input my gender$/) do
+  expect(page).to have_css('#your_details_form_gender')
+end
+
+And(/^I want to be able to input my salary$/) do
+  expect(page).to have_css('#your_details_form_salary')
```

Comment:
@margOnline as suggested by @tomas-stefano you can do something like `expect(@landing_page.salary).to_not be_empty`. Your code above is fine, but you can exploit some of the advantages of the page object for `@landing_page`.